packages/muelu/test/interface/ParameterListInterpreter.cpp

```diff
@@ -302,7 +302,7 @@ int main_(Teuchos::CommandLineProcessor &clp, Xpetra::UnderlyingLib& lib, int ar
           run_sed("'s/" + classes[q] + "<.*>/" + classes[q] + "<ignored> >/'", baseFile);
 
         // Strip ParamterList pointers
-        run_sed("'s/Teuchos::RCP<Teuchos::ParameterList const>{.*}/Teuchos::RCP<Teuchos::ParameterList const>{<ignored>} >/'", baseFile);
+        run_sed("'s/Teuchos::RCP<Teuchos::ParameterList const>{.*}/Teuchos::RCP<Teuchos::ParameterList const>{<ignored>}/'", baseFile);
```

Comment:
What was the problem with ">"?

Reply:
Before `Teuchos::RCP<Teuchos::ParameterList const>{ptr=0x296cf58,node=0x286dbd0,strong_count=2,weak_count=0}` was replaced by
`Teuchos::RCP<Teuchos::ParameterList const>{<ignored>} >`. The last `>` was not matching anything, but since this replacement happened on both the gold file and the test output, there was no error.
However, when using the filtered test output as the gold file, the closing `>` of `<ignored>` is matched, and we end up with two `>` in the filtered gold file, but only one in the filtered output file.